ContactDetailsApi/V1/Controllers/ContactDetailsController.cs

```diff
@@ -69,8 +70,9 @@ public async Task<IActionResult> GetContactDetailsByTargetId([FromQuery] Contact
         [LogCall(LogLevel.Information)]
         public async Task<IActionResult> DeleteContactDetailsById([FromQuery] DeleteContactQueryParameter queryParam)
         {
+            var token = _tokenFactory.Create(_httpContextWrapper.GetContextRequestHeaders(HttpContext));
 
-            var contact = await _deleteContactDetailsByTargetIdUseCase.Execute(queryParam).ConfigureAwait(false);
+            var contact = await _deleteContactDetailsByTargetIdUseCase.Execute(queryParam, token, ContactDetailsConstants.DELETED).ConfigureAwait(false);
```

Comment:
Not sure why you pass the event type constant from here. Surely you can not put it here and just specify in the use case (as each use case is specific to the event type)?

Reply:
Good point, I can revert that.